frontend/src/app/assess/[projectId]/premium-domains/page.tsx

```
@@ -473,9 +469,38 @@ export default function PremiumDomainsPage() {
+          <h1 className="text-2xl font-bold text-gray-900 dark:text-white mb-4">
+            No Premium Domains Available
+          </h1>
+          <p className="text-gray-600 dark:text-gray-300 mb-8">
+            There are currently no premium domains available for this project. Please check back later.
+          </p>
+          <div className="flex flex-col gap-4">
+            {projectId && (
+              <button
+                onClick={() => router.push(`/assess/${projectId}/fairness-bias/options`)}
+                className="flex items-center justify-center gap-2 px-6 py-3 bg-gradient-to-r from-purple-600 to-violet-600 hover:from-purple-700 hover:to-violet-700 text-white rounded-xl transition-all duration-300 font-semibold"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)